galaxy/templates/cronjob-maintenance.yaml

```diff
@@ -7,7 +7,11 @@ metadata:
   labels:
     {{- include "galaxy.labels" $ | nindent 4 }}
 spec:
+  {{- if $cronjob.enabled }}
   schedule: {{ $cronjob.schedule | quote }}
+  {{- else }}
```

Comment:
```suggestion
  {{- else }}
  # Set an impossible date so that the cronjob is still defined, but effectively disabled. 
  #  This will allow the cronjob to be run manually if needed.
```

Reply:
An example of how to run it manually using kubectl or similar would also be helpful I think

Reply:
There is an example in the README. Do we need one here as well?

Reply:
I had missed that - all good then.